src/modules/configs/schemas/application.py

```diff
@@ -0,0 +1,89 @@
+application = {
```

Comment:
Schemas related to each conf type should be committed (fixed up to or squashed) with commits introducing classes for these conf types e.g. `application` with `ApplicationConf`.